### DISCUSSION TITLE
PeaZip 7.5.0.0 Installer failed with exit code: 1

### DISCUSSION THREAD

**Discussion by peazip** [microsoft/winget-pkgs#5233] [Q&A]
Hello, I've updated the package for PeaZip (last version, 7.5.0.0) a few days ago. 
Validation completed successfully and "winget show peazip" now correctly find the up to date 7.5.0.0 version.
The problem is that "winget install peazip" ends in error: a graphic dialog (form the installer) is shown saying "access denied" and when the dialog is closed clicking on OK winget reports " Installer failed with exit code: 1 "

Edit: PeaZip installer is not signed (as in previous versions), and winget install does not prompts UAC screen, if this can be relevant.
Edit: The issue has been described also in https://github.com/microsoft/winget-cli/issues/199 (for older version of PeaZip) and https://github.com/microsoft/winget-cli/issues/189 (for many other applications)

**Comment by piraces** [comment]
It works for me in the preview version of winget-cli `v0.2.3162`.
It installs correctly without user input nor errors, prompting the UAC screen correctly.

**Comment by peazip** [comment]
On my test machine (Windows 10 20H2, 19042.630) last version reported as Windows Package Manager v0.2.3162 fails with same behavior and same error "Installer failed with exit code: 1" as in previous ones I tried.
